aws/multi_region.go

```diff
@@ -227,150 +211,3 @@ func listRegions(ctx context.Context, d *plugin.QueryData) (RegionsData, error)
 	d.ConnectionManager.Cache.Set(cacheKey, data)
 	return data, err
 }
-
-func unique(stringSlice []string) []string {
-	keys := make(map[string]bool)
-	list := []string{}
-	for _, entry := range stringSlice {
-		if _, value := keys[entry]; !value {
-			keys[entry] = true
-			list = append(list, entry)
-		}
-	}
-	return list
-}
-
-func SupportedRegionsForService(ctx context.Context, d *plugin.QueryData, serviceId string) ([]string, error) {
```

Comment:
@LalitTurbot Can you please summarize all of the changes made in `common_columns.go`, `multi_region.go`, `service.go`, and `service_v2.go` around checking for invalid regions, getting region lists for services, and listing service quotas services? I see a lot of functions moving, being renamed, etc., so I want to make sure that I understand the intent + scope of changes, thanks!

Reply:
It seems like you're pulling in a lot of changes made in https://github.com/turbot/steampipe-plugin-aws/pull/1296, are these changes critical for the migration of the tables in this PR?